src/components/select/select-list/select-list.component.js

```diff
@@ -82,12 +83,16 @@ const SelectList = React.forwardRef(
     const { blockScroll, allowScroll } = useScrollBlock();
 
     useEffect(() => {
-      blockScroll();
+      if (isOpen) {
+        blockScroll();
+      }
 
       return () => {
-        allowScroll();
+        if (isOpen) {
+          allowScroll();
+        }
```

Comment:
maybe? As I recall, I had to stop `blockScroll` being called if `isOpen` is false, otherwise scroll gets blocked when it shouldn't be (took me quite a while to figure out why the storybook page seemed to have broken completely :D). This should mean it's safe to not call `allowScroll` when tidying up in this case, but if there's no harm in calling it when scroll isn't blocked already then I can add it - I'll check.